src/app/shared/components/collection-card/collection-card.component.ts

```diff
@@ -21,7 +21,7 @@ export class CollectionCardComponent implements OnInit {
       this.collection.abvName !== 'plan c'
     ) {
       this.pictureLocation =
-        '../../../assets/images/collections/' +
+        '/assets/images/collections/' +
```

Comment:
This is a relative path instead of an absolute path. Just curious was anything updated for typescript?

Reply:
i don't know of any updates to typescript, but angular has a sourceRoot set to 'src', and `"assets": "src/assets"` is set in the angular.json. For imports you probably need absolute, but for images angular will accept relative path.